org.geeqie.Geeqie.json

```diff
@@ -94,21 +95,21 @@
             ]
         },
         "shared-modules/intltool/intltool-0.51.json",
-        "shared-modules/gtk2/gtk2.json",
         {
             "name": "geeqie",
             "config-opts": [
                 "--disable-lirc",
                 "--disable-lua",
                 "--disable-gpu-accel",
                 "--disable-ffmpegthumbnailer",
-                "--disable-gtk3"
+                "--enable-gtk3"
             ],
             "cleanup": [
                 "/share/man"
             ],
             "post-install": [
-                "install -Dm644 src/icons/svg/gqview_icon.svg /app/share/icons/hicolor/scalable/apps/geeqie.svg",
+                "install -Dm644 org.geeqie.Geeqie.appdata.xml /app/share/metainfo/org.geeqie.Geeqie.appdata.xml",
+                "install -Dm644 /app/share/pixmaps/geeqie.png /app/share/icons/hicolor/48x48/apps/geeqie.png",
```

Comment:
Why did you change the icon?

Reply:
I was naively undoing the commit from GTK3 -> GTK2. Will change that. Thanks.